src/Simplex/Chat/Migrations/M20221019_chat_unread_status.hs

```diff
@@ -0,0 +1,14 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20221019_chat_unread_status where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20221019_chat_unread_status :: Query
+m20221019_chat_unread_status =
+  [sql|
+ALTER TABLE contacts ADD COLUMN chat_unread INTEGER CHECK (chat_unread IN (NULL, 0, 1);
+ALTER TABLE groups ADD COLUMN chat_unread INTEGER CHECK (chat_unread IN (NULL, 0, 1);
```

Comment:
```suggestion
PRAGMA ignore_check_constraints=ON;

ALTER TABLE contacts ADD COLUMN chat_unread INTEGER CHECK (chat_unread NOT NULL);
UPDATE contacts SET chat_unread = 0;

ALTER TABLE groups ADD COLUMN chat_unread INTEGER CHECK (chat_unread NOT NULL);
UPDATE groups SET chat_unread = 0;

PRAGMA ignore_check_constraints=OFF;
```

This way you won't need to deal with Maybe when reading from the table, and any non-0 value will be read as True, so no need to constrain to 0 and 1.